shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/local/datasource/TokenLocalDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.avishek.phong.kmmic.data.local.datasource
+
+import co.nimblehq.avishek.phong.kmmic.data.remote.model.TokenApiModel
+import com.russhwolf.settings.ExperimentalSettingsApi
+import com.russhwolf.settings.Settings
+import com.russhwolf.settings.serialization.decodeValueOrNull
+import com.russhwolf.settings.serialization.encodeValue
+import kotlinx.serialization.ExperimentalSerializationApi
+
+private const val ACCESS_TOKEN_KEY = "access_token"
```

Comment:
Could we use `KEY` as a prefix? This is a common pattern we follow for snake case properties and resources.
```suggestion
private const val KEY_ACCESS_TOKEN = "access_token"
```

Reply:
solved in commit a2a4693